walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,7 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
+    name = name.replaceAll("[^A-Za-z]+", "");
```

Comment:
I think this works for the given tests :) but if they name "Ada Lovelace" we'd end up with "AdaLovelace".